src/main/java/com/inventor/email/EmailApp.java

```diff
@@ -0,0 +1,19 @@
+package com.inventor.email;
+
+import com.inventor.email.settings.EmailSettings;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+@SpringBootApplication
+public class EmailApp {
+
+    public static void main(String[] args) {
+        ApplicationContext content = SpringApplication.run(EmailApp.class, args);
+        InputConsole console = content.getBean(InputConsole.class);
+        EmailSettings making = content.getBean(EmailSettings.class);
+        making.addEmail(console.structureConsole());
+        System.out.println(making.getAll());
```

Comment:
It is better to use a logger here. It improves visibility and filtering of messages based on their importance. Please see here https://www.baeldung.com/spring-boot-logging